src/alfasim_sdk/default_tasks.py

```diff
@@ -240,16 +240,21 @@ def install_plugin_to_user_folder(ctx):
     try:
         hm_plugin_path = next(plugin_folder.glob("*.hmplugin"))
     except StopIteration:
-        raise RuntimeError(
-            f"Something went wrong on plugin compilation: "
-            f'could not find the "*.hmplugin" in {plugin_folder}.'
+        print_message(
+            f"Could not find any '*.hmplugin' in {plugin_folder}.",
+            color=Fore.RED,
+            bright=True,
         )
+        exit(0)
```

Comment:
Why exit with code `0`?
`0` use to represent no error took place. Is that not an error?

Also, prefer `raise invoke.Exit(code=0)`, or `raise invoke.Exit(msg, code=0)` (`msg` is printed to stderr)

Reply:
I think that when using `raise invoke.Exit(code=1)` invoke does not print a stack trace to the console before exiting

Reply:
Good suggestion. 

I didn't put much attention to the code `0`, indeed. I think a different value is a better way to handle it. Which one do you guys suggest?

However, I opted to use a direct `exit()` call instead raising a exception, because I was having some issues with the test of the task. The `invoke` is called using `subprocess.run()` and I even tough I was able to mock the `subprocess.run()` call, the coverage was not marking it as covered.

Reply:
Just changed the error code to 1.

Reply:
@prusse-martin Can you double check if this is acceptable, please?

Reply:
Actually, changing to `raise invoke.Exit(code=1)` didn't break my test 😅. So, thanks for the suggestion.